src/browser/synthesis-ui.js

```diff
@@ -46,18 +56,12 @@
   const clearInput = edit('input', Editable.clear);
 
 
+  const selectViewByIndex = (state, index) =>
```

Comment:
Nope, notice that `WebView.activate(...)` it makes `x.selected = x.previewed` and we set `x.previewed` so that two don't get out of sync when you for instance click a card to select it.